lib/Models/AbsIttCatalogItem.js

```diff
@@ -25,7 +25,8 @@ var proxyCatalogItemUrl = require('./proxyCatalogItemUrl');
 var AbsDataset = require('./AbsDataset');
 var AbsConcept = require('./AbsConcept');
 var AbsCode = require('./AbsCode');
-
+var naturalSort = require('javascript-natural-sort');
+naturalSort.insensitive = true;
```

Comment:
You need to do this right before it is used.  Otherwise, someone may have switched it back to case-sensitive sorting in the meantime.